scripts/docker_setup.sh

```diff
@@ -154,11 +211,12 @@ fi
 docker run --name bess -td --restart unless-stopped \
 	--cpuset-cpus=12-13 \
 	--ulimit memlock=-1 -v /dev/hugepages:/dev/hugepages \
+	-v /lib/firmware/intel:/lib/firmware/intel \
 	-v "$PWD/conf":/opt/bess/bessctl/conf \
 	--net container:pause \
 	$PRIVS \
 	$DEVICES \
-	upf-epc-bess:"$(<VERSION)" -grpc-url=0.0.0.0:$bessd_port
+	upf-epc-bess:"$(<VERSION)" -grpc-url=0.0.0.0:$bessd_port -m 2048
```

Comment:
Why is `-m 2048` needed?

Reply:
Needed for CNDP for more worker threads which uses more AF-XDP UMEMs. By default if no value is specified, per-socket hugepages used by BESS daemon is 1024. BESS daemon creates default DPDK Packet pool at startup time. CNDP allocates AF_XDP UMEM memory using DPDK Packet pool and it creates separate DPDK Mempool for each UMEM based on UMEM size.  If you prefer, we can use a variable with default value set as 1024 and can initialize to 2048 if mode is CNDP

Reply:
Given that this option is only needed for CNDP, what about doing something like this? or does `-m 2048` need to be at the end?
```patch
elif [ "$mode" == 'af_xdp' ]; then
	PRIVS='--privileged'

+elif [ "$mode" == 'cndp' ]; then
+	PRIVS='--privileged -m 2048'
...
...
	$PRIVS \
	$DEVICES \
	upf-epc-bess:"$(<VERSION)" -grpc-url=0.0.0.0:$bessd_port
```

Alternatively, you can try to use a variable (e.g., `CNDP_FLAG='-m 2048'`), but you need to make sure that it properly works for all modes
```patch
+if [ "$mode" == 'cndp' ]; then
+	CNDP_FLAG='-m 2048'
...
...
	$PRIVS \
	$DEVICES \
-	upf-epc-bess:"$(<VERSION)" -grpc-url=0.0.0.0:$bessd_port -m 2048
+	upf-epc-bess:"$(<VERSION)" -grpc-url=0.0.0.0:$bessd_port $CNDP_FLAG
```

Reply:
Done

Reply:
I prefer using HUGEPAGE variable in the script since it makes it more user friendly when testing with different modes (CNDP,DPDK) without making changes to the script. We can expand expand the [INSTALL.md](https://github.com/omec-project/upf/blob/master/docs/INSTALL.md) to add a section about Configuration: CNDP mode